templates/partials/_video.html

```diff
@@ -1,6 +1,6 @@
 {% if video.type == "youtube" %}
   <iframe id="ytplayer" type="text/html" width="818" height="460"
-          src="{{ video.url }}?autoplay=1&mute=1&loop=1&modestbranding=1&playlist={{ video.id }}" frameborder="0"></iframe>
+          src="{{ video.url }}?autoplay=1&mute=1&modestbranding=1&playlist={{ video.id }}" frameborder="0"></iframe>
```

Comment:
Not sure why `playlist` param is there at all, but it seems this is making video to repeat (even with loop turned off).